### ISSUE TITLE
!!Not really a issue!! FPS results

### ISSUE THREAD

**Issue by andrew-pa** [RolandMunsil/PixelWindow#1]
I grabbed your code and copied my Mandelbrot code into it, got some FPS data to compare with mine. (actually measured in seconds per frame because it makes more sense [SPF is bigger, program is slower linearly]) Very interesting, mine is vastly more complicated but for only a few microseconds more speed and not much more than that
See: https://onedrive.live.com/redir?resid=7C35F1F46ACF1745!5346&authkey=!ACpBt4F5Lc-Id6A&ithint=file%2cxlsx

**Comment by RolandMunsil** [comment]
Huh, interesting.

**Comment by andrew-pa** [comment]
Yeah, it wasn't very exact in being exact same code, do you want me to fork your project and upload my changes? I just copied my timer class over, so that could probably move in to a different header if you wanted to keep it

**Comment by RolandMunsil** [comment]
Sure, that would be nice.
